src/App.tsx

```diff
@@ -11,20 +13,25 @@ function App() {
     quoteData.quote5,
   ];
   return (
-    <div id="quoteContainer">
-      {qoutes.map((quote) => {
-        return (
-          <Quote
-            key={quote.id}
-            content={quote.content}
-            quoted={quote.quoted}
-            author={quote.author}
-            context={quote.context}
-            timestamp={quote.timestamp}
-          />
-        );
-      })}
-    </div>
+    <>
+      <div id="Header">
```

Comment:
This div could be removed. In Header.tsx you wrap the header with a div with the id "headerContainer". It is no need for two divs with different ids, when one of them only wraps around the other. Therefore, I think this div should be removed, so only the <Header />-component is present in this file.